x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel.tsx

```diff
@@ -635,6 +636,7 @@ export const VisualizationWrapper = ({
       <ExpressionRendererComponent
         className="lnsExpressionRenderer__component"
         padding="m"
+        variables={{ searchId: generateId() }}
```

Comment:
Do we need this? Seems like it's not necessary at the moment.

Reply:
Right now it's not required. I thought as a free addition for when we introduce the inspector in the editor, and have it here already. I see no harm in it.

Reply:
i would remove it until needed. i am not a huge fan for using variables for this.